src/common/filter/FunctionManager.cpp

```diff
@@ -272,7 +274,7 @@ FunctionManager::FunctionManager() {
         auto &attr = functions_["length"];
         attr.minArity_ = 1;
         attr.maxArity_ = 1;
-        attr.body_ = [] (const auto &args) {
+        attr.body_ = [] (const auto &args, const auto*) {
```

Comment:
why change to pointer？Does the reference have some problem ?

Reply:
It is timezone，mostly function doesn't need it.